src/settings-ui/Microsoft.PowerToys.Settings.UI.Library/ViewModels/PowerPreviewViewModel.cs

```diff
@@ -109,6 +111,24 @@ public bool MDRenderIsEnabled
             }
         }
 
+        public bool PDFRenderIsEnabled
+        {
+            get
+            {
+                return _pdfRenderIsEnabled;
+            }
+
+            set
+            {
+                if (value != _pdfRenderIsEnabled)
+                {
+                    _mdRenderIsEnabled = value;
```

Comment:
This will break the markdown renderer setting (!)

```suggestion
                    _pdfRenderIsEnabled = value;
```

Reply:
This is the one I fixed in https://github.com/rdeveen/PowerToys/pull/5